db/schema.rb

```diff
@@ -0,0 +1,34 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20190328140326) do
+
+  create_table "posts", force: :cascade do |t|
+    t.text     "content"
+    t.integer  "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
You don't have any foreign_key constraint for your post model. Kindly add one. 

Refer to this [article](https://thoughtbot.com/blog/referential-integrity-with-foreign-keys) for guidance on why and how to do so.

Reply:
Do you see this t.integer  "user_id"?

Reply:
Yup I see it. But that's hardly what I meant with my comment.